mon-pix/config/targets.js

```diff
@@ -10,7 +10,7 @@ const isCI = Boolean(process.env.CI);
 const isProduction = process.env.EMBER_ENV === 'production';
 
 if (isCI || isProduction) {
-  browsers.push('ie 9');
+  browsers.push('ie 11');
```

Comment:
Je lance quand même une question sur le sujet.
Mettre ie9 nous assurait de gérer les vieux navigateurs. En ce qui concerne les autres navigateurs (firefox, chrome), on se permettait de prendre seulement la dernière version, car la gestion d'ie9 nous permettait d'assurer que les anciennes versions de firefox étaient aussi gérées.
Hors, on a pu voir, surtout coté xp-eval, des soucis de bug sur des vieilles versions de firefox. Par chance aussi présente sur ie11.

Je me demande donc : 
- Est-ce qu'il ne faudrait pas être plus large sur les autres navigateurs si on passe ie à sa version 11.

De même, je sais que les résultats des navigateurs coté Matomo sont rassurants sur ce point.
**MAIS** nous souhaitons aussi bien maintenir les anciens navigateurs (et pour l'instant ie11), car avec l'ouverture à l'international (et notamment à certains pays), nous ne savons pas du tout ce à quoi va ressembler leur poste. 
Se baser uniquement sur les 1% de navigation sous ie actuelle n'est pas parlant par rapport aux utilisateurs que nous aurons dans les mois à venoir.

Reply:
Pour ma part je trouve que laisser IE 11 est largement suffisant surtout que pour rappel : [Microsoft ne le supporte plus](https://techcommunity.microsoft.com/t5/microsoft-365-blog/microsoft-365-apps-say-farewell-to-internet-explorer-11-and/ba-p/1591666). De plus, je ne vois pas comment ça peut fonctionner sachant qu'[Ember ne supporte plus IE 9 depuis la 3.0](https://blog.emberjs.com/ember-3-0-released/#browsersupportin30)

Reply:
> Est-ce qu'il ne faudrait pas être plus large sur les autres navigateurs si on passe ie à sa version 11 ?

Perso, je pense que oui, à chaque fois je suis surpris de voir qu'on ne supporte que la dernière version de chaque navigateur moderne.
Ce serait plus simple d'être explicite, et au moins on pourra dire facilement aux partenaires les versions supportées.

Reply:
Peut être autoriser au moins la version ESR de firefox  ?

Reply:
Je suis inquiet aussi de voir ces `last 1 versions` pour les navigateurs "modernes", on pourrait effectivement être plus explicite. Je ne crois pas cela dit que le passage de IE9 à IE11 comme "plancher" change grand-chose au support des autres navigateurs.